tests/yam/agama/boot_agama.pm

```diff
@@ -53,7 +53,11 @@ sub run {
     my $agama_up_an_running = $testapi::distri->get_agama_up_an_running();
 
     # prepare kernel parameters
+    if (my $agama_profile = get_var('AGAMA_PROFILE')) {
```

Comment:
the comment above should be after this new `if` block.

Reply:
Updated.